dtschema/meta-schemas/nvmem.yaml

```diff
@@ -10,7 +10,9 @@ properties:
     $ref: "cell.yaml#/array"
 
   nvmem-cells:
-    $ref: "cell.yaml#/array"
+    oneOf:
+      - $ref: "cell.yaml#/array"
+      - $ref: "/schemas/types.yaml#/definitions/phandle-array"
```

Comment:
meta-schemas are what check the schemas. What you are adding is what belongs in the schema, and is already there (in the kernel tree).